controllers/managedfusionoffering_controller.go

```diff
@@ -190,6 +194,15 @@ func (r *ManagedFusionOfferingReconciler) reconcilePhases() (reconcile.Result, e
 
 }
 
+func reconcileOfferingConfig(offering *v1alpha1.ManagedFusionOffering) error {
+
+	offeringConfigYaml := map[interface{}]interface{}{}
```

Comment:
Just for my understanding, why `map[interface{}]interface{}{}` and not `interface{}`?

Reply:
My bad, I din't realise we are not going to parse the spec here or iterate within the elements  but only validating the yaml.
Changed the implementation to use interface{}